src/foam/u2/filter/BooleanFilterView.js

```diff
@@ -0,0 +1,133 @@
+/**
+* @license
+* Copyright 2019 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.u2.filter',
+  name: 'BooleanFilterView',
+  extends: 'foam.u2.Controller',
+
+  documentation: `
+
+  `,
+
+  requires: [
+    'foam.mlang.predicate.True',
+    'foam.u2.CheckBox'
+  ],
+
+  css: `
+    ^container {
+      display: flex;
+      align-items: center;
+      padding: 4px 16px;
+    }
+
+    ^container:hover {
+      cursor: pointer;
+      background-color: #f5f7fa;
+    }
+
+    ^container:first-child {
+      margin-top: 20px;
+    }
+
+    ^container:last-child {
+      margin-bottom: 20px;
+    }
+
+    ^container .foam-u2-md-CheckBox-label {
+      position: relative;
+      margin-top: 0;
+    }
+
+    ^container .foam-u2-md-CheckBox {
+      border-color: #9ba1a6;
+    }
+
+    ^container .foam-u2-md-CheckBox:checked {
+      background-color: #406dea;
+      border-color: #406dea;
+    }
+  `,
+
+  properties: [
+    {
+      name: 'property',
+      documentation: `The property that this view is filtering by. Should be of
```

Comment:
For this and the others, is this requirement enforced anywhere?  What does configuration look like for developers - what is required in the model or table views?

Reply:
`property` right now is being populated by the property defined in `searchColumns` in a model. So like before, if you define a property in `searchColumns`, it will be used as a filterable parameter.

As far as my knowledge goes, it is not enforced atm